addons/website/controllers/backend.py

```diff
@@ -36,3 +38,13 @@ def fetch_dashboard_data(self, website_id, date_from, date_to):
     @http.route('/website/iframefallback', type="http", auth='user', website=True)
     def get_iframe_fallback(self):
         return request.render('website.iframefallback')
+
+    @http.route('/website/check_new_content_access_rights', type="json", auth='user')
+    def check_create_access_rights(self, models):
+        if not request.env.user.has_group('website.group_website_restricted_editor'):
```

Comment:
Add a docstring comment for this controller explaining we should probably find a better way in master. Should we choose to leave it that way in master for now, it's cool to mark things that should be improved IMO.